application/src/main/kotlin/com/studentcenter/weave/application/meeting/service/application/CreateMeetingAttendanceService.kt

```diff
@@ -94,7 +97,11 @@ class MeetingAttendanceCreateApplicationService(
         meeting: Meeting,
         memberCount: Int,
     ) {
-        meetingDomainService.save(meeting.complete())
+        meeting
+            .complete()
+            .also { meetingDomainService.save(it) }
+            .createCompletedEvent()
+            .also { meetingEventPublisher.publish(it) }
```

Comment:
p3) 이렇게는 별로인가요? meeting이 complete 되면 저장과 이벤트 발행이 모두 일어나니까?
```suggestion
            .also { 
            	meetingDomainService.save(it) 
            	 meetingEventPublisher.publish(it.createCompletedEvent())
            }
```

Reply:
p3) 아니면 기므보니 여기서 어떤 이벤트를 발행할지 메서드명으로 하는 건 어떤가요?
```suggestion
            .also { 
              meetingDomainService.save(it) 
              meetingEventPublisher.publishCompletedMeeting(it)
            }
```

Reply:
meeting으로 일괄적으로 처리하면 좋겠지만, 이벤트 종류마다 도메인 외에도 추가적으로 필요한 데이터가 있을수도 있어서 전자가 더 좋을것 같아요!